src/util/system.h

```diff
@@ -264,6 +264,16 @@ class ArgsManager
      */
     std::optional<const Command> GetCommand() const;
 
+    /**
+     * Get a normalized path from a specified pathlike argument
+     *
+     * It is guaranteed that the returned path has no trailing slashes.
+     *
+     * @param pathlike_arg Pathlike argument to get a path from (e.g., "-datadir", "-blocksdir" or "-walletdir")
+     * @return Normalized path which is get from a specified pathlike argument
+     */
+    fs::path GetPathArg(std::string pathlike_arg) const;
```

Comment:
Other `Get...Arg` methods use the camelCase notation for arguments. Let's change to `pathlikeArg` or similar.

Reply:
For new code we strive to follow our [symbol naming convention](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c):
> Variable (including function arguments) and namespace names are all lowercase and may use _ to separate words (snake_case).